src/EnergyPlus/DualDuct.cc

```diff
@@ -1431,7 +1273,7 @@ namespace DualDuct {
         // The calculated load from the Heat Balance
         QTotLoad = state.dataZoneEnergyDemand->ZoneSysEnergyDemand(ZoneNum).RemainingOutputRequired;
         // Calculate all of the required Cp's
-        CpAirZn = PsyCpAirFnW(state.dataLoopNodes->Node(ZoneNodeNum).HumRat);
+        CpAirZn = Psychrometrics::PsyCpAirFnW(state.dataLoopNodes->Node(ZoneNodeNum).HumRat);
```

Comment:
As explicitly resolved names start to proliferate, I suggest we start to migrate to fewer `namespace`s and given them names like `Psych` and `Sched`.